server/plugins/TriggerUGens.cpp

```diff
@@ -1459,9 +1462,14 @@ void Sweep_next_kk(Sweep* unit, int inNumSamples) {
     if (previn <= 0.f && curin > 0.f) {
```

Comment:
What do you think about renaming the variable `rate` to `step` or `stepsize` in cases where the `rate` arg is control rate. After the mul by `SAMPLEDUR`, that value is not longer a "rate". It may even read better if all instances of `rate` are called `step`.